application/extensions/admin/survey/LanguagesWidget/LanguagesWidget.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+class LanguagesWidget extends WhSelect2
```

Comment:
Hm, why is a new widget needed for a simple select box? Or did I miss something?

Reply:
And why is that needed?

Reply:
Ah ok. That's a good answer. :)

Reply:
> Ah ok. That's a good answer. :)

I have my moments :)